src/minesweeper/Config.py

```diff
@@ -0,0 +1,10 @@
+
+import pygame
+from ctypes.wintypes import RGB
+
+class Config:
+    def __init__(self):
+        self.screen_width = 120
+        self.screen_height = 80
+        self.bg_color = RGB(0, 0, 0)
+        self.clock = pygame.time.Clock()
```

Comment:
The configuration should not have runnable code like `Clock()`. Instead, save the FPS as `60`, for example,

```python
self.FPS = 60
```

and use `clock = pygame.time.Clock()` in `runGame` together with `clock.tick(config.FPS)`.